src/OpenTelemetry/Metrics/AggregatorStore.cs

```diff
@@ -56,19 +54,19 @@ internal IAggregator[] MapToMetrics(string[] seqKey, object[] seqVal)
             // TODO: Need to map each instrument to metrics (based on View API)
             if (this.instrument.GetType().Name.Contains("Counter"))
             {
-                aggregators.Add(new SumMetricAggregator(name, this.instrument.Description, this.instrument.Unit, this.instrument.Meter, dt, tags));
+                aggregators.Add(new SumMetricAggregator(this.instrument.Name, this.instrument.Description, this.instrument.Unit, this.instrument.Meter, dt, tags));
```

Comment:
unrelated, but a minor optimization, as Meter is anyway available in the exported item, and exporters may get its name, if it chose to.